beacon-chain/blockchain/fork_choice.go

```diff
@@ -105,24 +115,50 @@ func (c *ChainService) updateFFGCheckPts(state *pb.BeaconState) error {
 
 // ApplyForkChoiceRule determines the current beacon chain head using LMD GHOST as a block-vote
 // weighted function to select a canonical head in Ethereum Serenity.
-func (c *ChainService) ApplyForkChoiceRule(ctx context.Context, block *pb.BeaconBlock, computedState *pb.BeaconState) error {
+func (c *ChainService) ApplyForkChoiceRule(ctx context.Context, block *pb.BeaconBlock, postState *pb.BeaconState) error {
 	ctx, span := trace.StartSpan(ctx, "beacon-chain.blockchain.ApplyForkChoiceRule")
 	defer span.End()
-	h, err := hashutil.HashBeaconBlock(block)
+	log.Info("Applying LMD-GHOST Fork Choice Rule")
```

Comment:
Should this be a "info" log? How often would it print out?

Reply:
Everytime fork choice is run, this is a part of #2050. Most of the changes are minor here, the diff for comparison https://github.com/prysmaticlabs/prysm/compare/testruntime...fixSync